packages/fast-components-react-base/src/auto-suggest/auto-suggest.tsx

```diff
@@ -274,6 +287,20 @@ class AutoSuggest extends Foundation<
         );
     };
 
+    /**
+     * Handle blur events
+     */
+    private handleBlurCapture = (event: React.FocusEvent): void => {
+        if (
+            this.state.isMenuOpen &&
+            this.rootElement.current !== null &&
```

Comment:
nit: just for consistency, I think it would be good to check the same way as above using isNil

Reply:
yep